packages/react/src/grid/GridCell.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { PropsWithChildren } from 'react'
+import { PageGridColumnNumber } from './PageGrid'
+
+export type GridCellProps = PropsWithChildren<{
+  gridColumns?: [PageGridColumnNumber, PageGridColumnNumber]
+}>
+
+export const gridCellClassNames = (gridColumns?: [PageGridColumnNumber, PageGridColumnNumber]) => {
+  if (!gridColumns) {
+    return undefined
+  }
+
+  const [start, end] = gridColumns
```

Comment:
Could start be 10 and end 5? Otherwise we might need some validation?

Reply:
We’ve avoided this now that we use `start` and `span` rather than `end`.